test/recipes/80-test_ssl_old.t

```diff
@@ -445,6 +443,8 @@ sub testssl {
                 $flag = "-ssl3";
             } elsif ($protocol eq "TLSv1.2") {
                 $flag = "-tls1_2";
+            } elsif ($protocol eq "TLSv1.3") {
```

Comment:
How about

We don't need to specify TLSv1.3 explicitly in a flag, as it's currently the highest supported protocol version and thus the default choice.